src/config.ts

```diff
@@ -90,37 +90,38 @@ const config = {
       certDuration: 31536000,
     },
     networkConfig: {
-      // ForwardProxy
-      forwardHost: '127.0.0.1' as Host,
-      forwardPort: 0 as Port,
-      proxyHost: '0.0.0.0' as Host,
-      proxyPort: 0 as Port,
       // GRPCServer for agent service
       agentHost: '127.0.0.1' as Host,
       agentPort: 0 as Port,
+      ipv6Only: false,
       // RPCServer for client service
       clientHost: '127.0.0.1' as Host,
       clientPort: 0 as Port,
-      // Times and limits for client communication
+      // Websocket server config
+      maxReadableStreamBytes: 1_000_000_000, // About 1 GB
```

Comment:
If I recall, it's not possible to properly apply backpressure to the websocket readable stream. This is large to avoid any blocking because that would cause data to be lost. I can review it again.

Reply:
According to these 2:

* https://stackoverflow.com/questions/19414277/can-i-have-flow-control-on-my-websockets
* https://stackoverflow.com/questions/55188328/does-nodejs-websockets-ws-module-implement-backpressure

Indeed browser websockets don't have backpressure built into it. I think simply because `send` does not return a promise: https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/send.

Chrome has a new `WebSocketStream` which is apparently chrome-only.

There is a workaround that involves checking the buffer size. They say it is inefficient... sure, but I think I prefer elegance here over some decrease in performance. Besides we don't even know how much lower the performance is, and with a big enough buffer size we can amortise the cost of checking the buffer size.

I think we should actually implement a workaround.

On the nodejs side with ws or whatever, the same workaround should be applicable or an alternative method.

Furthermore I'm beginning to think uws is a mistake, we would want a proper runtime-less websocket library in the future which we'll put into `js-ws` anyway so what's the point of having both `ws` and `uws` in PK right now... it's kind of pointless.

Reply:
![image](https://github.com/MatrixAI/Polykey/assets/640797/8d366df0-ba0d-4ab6-8ff3-3ca29400bf36)

Reply:
While browser websocket doesn't have the `callback` option. The `ws` library does in fact provide a callback:

https://github.com/websockets/ws/blob/master/doc/ws.md#websocketsenddata-options-callback

This basically means the API between browser and server will deviate a bit. You could use the callback as explained in https://stackoverflow.com/questions/55188328/does-nodejs-websockets-ws-module-implement-backpressure to implement backpressure.

But on the client side one would have to check the buffer size.

Reply:
The epic can also talk about removing uws since it's just adding extra complexity and no bang for our buck (cross-platformness will end up requiring runtimeless state machine like js-quic anyway).

Reply:
Created an epic for tracking `websocket` changes at #541